libs/core/control.ts

```diff
@@ -1,12 +1,15 @@
 /**
-* Runtime and event utilities.
+* Program controls and events.
 */
 //% weight=10 color="#31bca3" icon="\uf110" advanced=true
 namespace control {
 
     /**
-     * Display specified error code and stop the program.
+     * Display an error code and stop the program.
+     * @param code an error number to display. eg: 5
      */
+    //% help=control/panic weight=29
+    //% blockId="control_panic" block="panic %code"
```

Comment:
Same thing here. see other PR for whether or not we want to surface panic as a block.